gnomad_qc/v3/resources/basics.py

```diff
@@ -0,0 +1,67 @@
+import hail as hl
+from gnomad.resources.resource_utils import (MatrixTableResource,
+                                             VersionedMatrixTableResource)
+
+from gnomad_qc.v3.resources.constants import CURRENT_RELEASE
+from gnomad_qc.v3.resources.meta import meta
+from gnomad_qc.v3.resources.sample_qc import hard_filtered_samples
+
+
+def get_gnomad_v3_mt(
+    split=False,
+    key_by_locus_and_alleles: bool = False,
+    remove_hard_filtered_samples: bool = True,
+    release_only: bool = False,
+    samples_meta: bool = False,
+) -> hl.MatrixTable:
+    """
+    Wrapper function to get gnomAD data with desired filtering and metadata annotations
+
+    :param split: Perform split on MT - Note: this will perform a split on the MT rather than grab an already split MT
+    :param key_by_locus_and_alleles: Whether to key the MatrixTable by locus and alleles (only needed for v3)
+    :param remove_hard_filtered_samples: Whether to remove samples that failed hard filters (only relevant after sample QC)
+    :param release_only: Whether to filter the MT to only samples available for release (can only be used if metadata is present)
+    :param samples_meta: Whether to add metadata to MT in 'meta' column
+    :return: gnomAD v3 dataset with chosen annotations and filters
+    """
+    mt = gnomad_v3_genotypes.mt()
+    if key_by_locus_and_alleles:
+        mt = hl.MatrixTable(
+            hl.ir.MatrixKeyRowsBy(mt._mir, ["locus", "alleles"], is_sorted=True)
```

Comment:
Why use this vs `mt.key_rows_by`? Might be worth a comment.

Reply:
The MT is already sorted by locus and here each locus only appears once so this prevents hail from running it's sorting algorithm to save on runtime. Will add the comment.